client/components/payment-activity/index.tsx

```diff
@@ -5,57 +5,94 @@ import * as React from 'react';
 import { Card, CardBody, CardHeader } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import interpolateComponents from '@automattic/interpolate-components';
+import moment from 'moment';
 
 /**
  * Internal dependencies
  */
 
 import EmptyStateAsset from 'assets/images/payment-activity-empty-state.svg?asset';
-import PaymentActivityData from './payment-activity-data';
+import PaymentActivityDataComponent from './payment-activity-data';
 import Survey from './survey';
 import { WcPayOverviewSurveyContextProvider } from './survey/context';
+import { usePaymentActivityData } from 'wcpay/data';
+import type { DateRange } from './types';
 import './style.scss';
 
+/**
+ * This will be replaces in the future with a dynamic date range picker.
+ */
+const getDateRange = (): DateRange => {
+	return {
+		// Subtract 7 days from the current date.
+		date_start: moment()
+			.subtract( 7, 'd' )
```

Comment:
This date difference was previously set to 6 days. I'm assuming this was intentional, but want to double-check with you.